src/main/java/org/jenkins/plugins/lockableresources/LockStepExecution.java

```diff
@@ -50,57 +50,95 @@ public boolean start() throws Exception {
 
         getContext().get(FlowNode.class).addAction(new PauseAction("Lock"));
         PrintStream logger = getContext().get(TaskListener.class).getLogger();
-        logger.println("Trying to acquire lock on [" + step + "]");
+
+        Run<?, ?> run = getContext().get(Run.class);
+        LockableResourcesManager.printLogs("Trying to acquire lock on [" + step + "]", Level.INFO, LOGGER, logger);
```

Comment:
Please reduce the log level of those kind of messages as it floods the Jenkins log on instances with many locks.